### PR TITLE
Dropdown: Headers and Dividers marked as hidden are no longer visible

### LINKED ISSUE #19909: [Dropdown] Headers and dividers marked as hidden, are still shown
### Environment Information

- **Package version(s)**: 7.170.3
- **Browser and OS versions**: Not relevant

### Please provide a reproduction of the bug in a codepen:

https://codepen.io/chmsft/pen/XWgBgmW

#### Actual behavior:

Marking dropdown items as hidden only works for normal items.  Headers and Dividers are not hidden, even though the prop is valid for them and documentation doesn't say they shouldn't be hidden.  

#### Expected behavior:

If I mark an item (regardless of type) as 'hidden=true', it should be hidden.

### Priorities and help requested:

Are you willing to submit a PR to fix? No

Requested priority: Normal

### PR DESCRIPTION
**Pull request by TristanWatanabe**
Please verify that:
* [x] Code is up-to-date with the `master` branch
* [x] Your changes are covered by tests (if possible)
* [x] You've run `yarn change` locally

## Current Behavior

- When an option item for a `Dropdown` are either a `header` or `divider` and given the `hidden:true` property, they are still visible:
image

## New Behavior

- `header` and `divider` `Dropdown` options are no longer visible when marked as `hidden`
image

## Related Issue(s)

Fixes #19909

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/a21bfaceecf201d0c984fa84704148a24f956901'>a21bfaceecf201d0c984fa84704148a24f956901</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Avatar-mount_base.html">986</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Avatar-mount.html">998</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/BaseButton-mount_base.html">961</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/BaseButton-mount.html">972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Breadcrumb-mount_base.html">2791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Breadcrumb-mount.html">2782</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ButtonNext-mount_base.html">540</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ButtonNext-mount.html">548</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Checkbox-mount_base.html">1622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Checkbox-mount.html">1587</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/CheckboxBase-mount_base.html">1341</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/CheckboxBase-mount.html">1371</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ChoiceGroup-mount_base.html">4924</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ChoiceGroup-mount.html">4920</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ComboBox-mount_base.html">1005</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ComboBox-mount.html">1038</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/CommandBar-mount_base.html">10311</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/CommandBar-mount.html">10576</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ContextualMenu-mount_base.html">8565</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ContextualMenu-mount.html">8585</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DefaultButton-mount_base.html">1175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DefaultButton-mount.html">1188</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DetailsRow-mount_base.html">3828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DetailsRow-mount.html">3802</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DetailsRowFast-mount_base.html">3918</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DetailsRowFast-mount.html">3845</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DetailsRowNoStyles-mount_base.html">3690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DetailsRowNoStyles-mount.html">3728</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Dialog-mount_base.html">2671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Dialog-mount.html">2623</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DocumentCardTitle-mount_base.html">190</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/DocumentCardTitle-mount.html">197</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Dropdown-mount_base.html">3280</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Dropdown-mount.html">3317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderNext-mount_base.html">1893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderNext-mount.html">1930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderWithTheme-mount_base.html">162</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderWithTheme-mount.html">166</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">123</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderWithTheme-virtual-rerender.html">118</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">205</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">217</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FocusTrapZone-mount_base.html">1946</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FocusTrapZone-mount.html">1886</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FocusZone-mount_base.html">1874</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/FocusZone-mount.html">1862</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/IconButton-mount_base.html">1786</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/IconButton-mount.html">1759</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Label-mount_base.html">385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Label-mount.html">374</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Layer-mount_base.html">3061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Layer-mount.html">3037</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Link-mount_base.html">508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Link-mount.html">528</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/MakeStyles-mount_base.html">1774</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/MakeStyles-mount.html">1792</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/MenuButton-mount_base.html">1537</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/MenuButton-mount.html">1499</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/MessageBar-mount_base.html">2083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/MessageBar-mount.html">2036</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Nav-mount_base.html">3263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Nav-mount.html">3402</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/OverflowSet-mount_base.html">1153</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/OverflowSet-mount.html">1164</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Panel-mount_base.html">2584</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Panel-mount.html">2555</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Persona-mount_base.html">883</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Persona-mount.html">908</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Pivot-mount_base.html">1461</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Pivot-mount.html">1489</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/PrimaryButton-mount_base.html">1353</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/PrimaryButton-mount.html">1336</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Rating-mount_base.html">7837</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Rating-mount.html">7817</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SearchBox-mount_base.html">1379</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SearchBox-mount.html">1417</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Shimmer-mount_base.html">2623</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Shimmer-mount.html">2604</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Slider-mount_base.html">1976</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Slider-mount.html">1968</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SpinButton-mount_base.html">5127</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SpinButton-mount.html">5058</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Spinner-mount_base.html">475</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Spinner-mount.html">452</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SplitButton-mount_base.html">3295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SplitButton-mount.html">3213</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Stack-mount_base.html">539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Stack-mount.html">560</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/StackWithIntrinsicChildren-mount_base.html">2350</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/StackWithIntrinsicChildren-mount.html">2330</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/StackWithTextChildren-mount_base.html">5265</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/StackWithTextChildren-mount.html">5410</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SwatchColorPicker-mount_base.html">11605</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/SwatchColorPicker-mount.html">11589</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/TagPicker-mount_base.html">2639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/TagPicker-mount.html">2755</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/TeachingBubble-mount_base.html">13390</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/TeachingBubble-mount.html">13266</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Text-mount_base.html">467</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Text-mount.html">481</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/TextField-mount_base.html">1417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/TextField-mount.html">1454</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ThemeProvider-mount_base.html">1265</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ThemeProvider-mount.html">1237</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ThemeProvider-virtual-rerender_base.html">693</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ThemeProvider-virtual-rerender.html">655</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1970</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1903</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Toggle-mount_base.html">841</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/Toggle-mount.html">856</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/buttonNative-mount_base.html">165</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21527/perf-test/buttonNative-mount.html">158</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

File: packages/react/src/components/Dropdown/Dropdown.styles.ts [hunk 1]
```diff
@@ -379,29 +404,8 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
...
-        textAlign: 'left',
-        selectors: {
-          [HighContrastSelector]: {
-            color: 'GrayText',
-            ...getHighContrastNoAdjustStyle(),
-          },
-        },
-      },
-    ],
+    dropdownItemHeader: [...dropdownHeaderStyle],
```
Comment (spmonahan) [inline packages/react/src/components/Dropdown/Dropdown.styles.ts:407]: nit: does `dropdownHeaderStyle` need to be spread here? Looks like the same usage as `dropdownItemSelected` on L388 and we're not using spread there.

**Review by spmonahan — APPROVED** [review]

File: packages/react/src/components/Dropdown/Dropdown.styles.ts (hunk 1 above)
Comment (TristanWatanabe) [reply packages/react/src/components/Dropdown/Dropdown.styles.ts]: Good point, a spread is not necessary here - just made the change!

**Comment by spmonahan** [comment]
LGTM

**Pull request merged**